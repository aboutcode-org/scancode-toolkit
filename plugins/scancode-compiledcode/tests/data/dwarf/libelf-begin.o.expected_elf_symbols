{
  "files": [
    "begin.c"
  ], 
  "external_libs_objects": [], 
  "global_functions": [
    "gelf_getclass", 
    "elf_memory", 
    "elf_begin"
  ], 
  "standard_files": [], 
  "local_functions": [
    "getnum", 
    "_elf_check_type"
  ], 
  "external_libs_functions": [], 
  "standard_objects": [], 
  "global_objects": [], 
  "standard_functions": [], 
  "local_objects": [
    "fmag", 
    "_elf_init"
  ]
}