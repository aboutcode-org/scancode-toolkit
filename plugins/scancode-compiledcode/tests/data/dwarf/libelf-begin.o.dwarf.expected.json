{
  "original_source_files": [
    "/home/jqbx34/djcd/libelf-0.8.10/lib/begin.c", 
    "/home/jqbx34/djcd/libelf-0.8.10/lib/libelf.h", 
    "/home/jqbx34/djcd/libelf-0.8.10/lib/private.h"
  ], 
  "included_source_files": [
    "/usr/include/stdint.h", 
    "/usr/lib/gcc/i586-suse-linux/4.2.1/include/stddef.h", 
    "/usr/include/bits/types.h", 
    "/usr/include/sys/types.h", 
    "/usr/include/time.h", 
    "/usr/include/elf.h", 
    "/usr/include/ar.h"
  ]
}